### PR TITLE
Amends and corrects some meh wording on the operating console.

### PR DESCRIPTION
**Pull request by Ragolution**
Changes confusing wording on Optable console

### CODE REVIEW

**Comment by deathride58** [comment]
It's up to the other maintainers whether they want this to be merged or not. This may lead to potential confusion from people who might think that the surgery is for erotic purposes and not for reviving a body.

**Comment by Ragolution** [comment]
HOW IS IT A MEME PR YOU LITERALLY NEED TO STICK YOUR BATON INTO THEM

wait fuck

File: code/modules/surgery/advanced/revival.dm [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	return TRUE
 
 /datum/surgery_step/revive
-	name = "repair body"
+	name = "electrically stimulate brain"
```
Comment (Ghommie) [inline code/modules/surgery/advanced/revival.dm:29]: Just change it to "revive body", doctor Frankenstein.

Comment (Ragolution) [reply code/modules/surgery/advanced/revival.dm:29]: ooga

**Review by deathride58 — APPROVED** [review]
I GUESS

**Pull request merged**